doc/rofi-theme.5.markdown

```diff
@@ -770,17 +769,17 @@ Explanation of the different parts:
  * Border - A border that goes around the padding and content.
    The border use the foreground color of the widget.
  * Margin - Clears an area outside the border.
-   The margin is transparent
+   The margin is transparent.
 
 The box model allows us to add a border around elements, and to define space between elements.
 
-The size, on each side, of margin, border and padding can be set.
-For the border a linestyle and radius can be set.
+The size of each margin, border, and padding can be set.
+For the border, a linestyle and radius can be set.
 
 ### Spacing
 
-Widgets that can pack more then one child widget, currently box and listview, the `spacing` property exists.
-This determines the space between the packed widgets (both in horizontal as vertical direction).
+Widgets that can pack more then one child widget (currently box and listview) have the `spacing` property.
```

Comment:
is correct, but maybe maybe still add:
This property sets the spacing between the children in both the horizontal and vertical direction.

(or is picture clear enough?)